profile.md

```diff
@@ -54,6 +59,75 @@ the [Open Service Broker API specification](spec.md) SHALL take precedence.
 
 - *Application*: The software that uses a Service Instance via Service Binding.
 
+## Originating Identity Header
+
+In the [Open Service Broker API specification](spec.md) it defines an
+additional HTTP Header that can be included in messages from the platform
+to identify the user that requested the action to be taken.
+
+The header consists of two parts: a `platform` string and `value` string.
+Both values will vary based on the platform which is being used. The
+following sections define the values to be used based on the Platform
+and which properties are expected to appear in the JSON encoded
+`value`.
+
+Note that when both the originating identity HTTP Header and the Context
+object appear in the same message the `platform` value MUST be the same
+for both.
+
+### Cloud Foundry
+
+*`platform` Value*: `cloudfoundry`
+
+The following properties MUST appear within the JSON encoded `value`:
```

Comment:
> That's a good point. I'm going to remove it since if its optional then people will need to deal with it being absent anyway so we might as well enforce consistency and just require user_id.

I see cases where having `user_name` is useful: this avoids requiring trusted brokers that need user identity to have credentials with each platform and lookup user identity from user id in a platform specific way ( e.g. http://docs.cloudfoundry.org/api/uaa/#lookup-user-ids-usernames for CF). But this comes with a cost on platform side to control access/visibility of which broker are trusted or not.